<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddFeedDialog</class>
 <widget class="QDialog" name="AddFeedDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>739</width>
    <height>653</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Create new feed</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>:/images/rstray3.png</normaloff>:/images/rstray3.png</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="HeaderFrame" name="headerFrame"/>
   </item>
   <item>
    <widget class="QFrame" name="gradFrame">
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QGridLayout" name="gradFrame_GL">
      <item row="7" column="0">
       <widget class="QGroupBox" name="authenticationGroupBox">
        <property name="title">
         <string>Authentication (not yet supported)</string>
        </property>
        <layout class="QGridLayout" name="gridLayout_4">
         <item row="0" column="0" colspan="2">
          <widget class="QCheckBox" name="useAuthenticationCheckBox">
           <property name="text">
            <string>Feed needs authentication</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="userLabel">
           <property name="text">
            <string>User</string>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="passwordLabel">
           <property name="text">
            <string>Password</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QLineEdit" name="userLineEdit"/>
         </item>
         <item row="2" column="1">
          <widget class="QLineEdit" name="passwordLineEdit">
           <property name="echoMode">
            <enum>QLineEdit::Password</enum>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="7" column="1">
       <widget class="QGroupBox" name="storageTimeGroupBox">
        <property name="title">
         <string>Storage time</string>
        </property>
        <layout class="QGridLayout" name="gridLayout_3">
         <item row="2" column="0">
          <widget class="QLabel" name="storageTimeLabel">
           <property name="text">
            <string>Days (0 = off)</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QSpinBox" name="storageTimeSpinBox">
           <property name="maximumSize">
            <size>
             <width>50</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="maximum">
            <number>999999999</number>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QCheckBox" name="useStandardStorageTimeCheckBox">
           <property name="text">
            <string>Use standard storage time</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QPushButton" name="clearCachePushButton">
           <property name="text">
            <string>Clear cache</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="8" column="1">
       <widget class="QGroupBox" name="proxyGroupBox">
        <property name="title">
         <string>Proxy</string>
        </property>
        <layout class="QGridLayout" name="gridLayout_6">
         <item row="0" column="0" colspan="2">
          <widget class="QCheckBox" name="useStandardProxyCheckBox">
           <property name="text">
            <string>Use standard proxy</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="serverLabel">
           <property name="text">
            <string>Server</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QLineEdit" name="proxyAddressLineEdit"/>
         </item>
         <item row="1" column="2">
          <widget class="QLabel" name="portLabel">
           <property name="text">
            <string>:</string>
           </property>
          </widget>
         </item>
         <item row="1" column="3">
          <widget class="QSpinBox" name="proxyPortSpinBox">
           <property name="maximum">
            <number>65535</number>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="8" column="0">
       <widget class="QGroupBox" name="updateInteralGroupBox">
        <property name="title">
         <string>Update interval</string>
        </property>
        <layout class="QGridLayout" name="gridLayout_5">
         <item row="0" column="0" colspan="2">
          <widget class="QCheckBox" name="useStandardUpdateInterval">
           <property name="text">
            <string>Use standard update interval</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="updateIntervalLabel">
           <property name="text">
            <string>Interval in minutes (0 = manual)</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QSpinBox" name="updateIntervalSpinBox">
           <property name="maximumSize">
            <size>
             <width>50</width>
             <height>16777215</height>
            </size>
           </property>
          </widget>
         </item>
         <item row="2" column="0" colspan="2">
          <layout class="QHBoxLayout" name="lastUpdateLayout">
           <item>
            <widget class="QLabel" name="lastUpdateLabel">
             <property name="text">
              <string>Last update</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="lastUpdate">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="text">
              <string>Never</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
      <item row="11" column="0" colspan="2">
       <layout class="QHBoxLayout" name="buttonLayout">
        <item>
         <widget class="QDialogButtonBox" name="buttonBox">
          <property name="standardButtons">
           <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="1" column="0" colspan="2">
       <layout class="QVBoxLayout" name="descriptionLayout">
        <item>
         <widget class="QLabel" name="descriptionLabel">
          <property name="text">
           <string>Description:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPlainTextEdit" name="descriptionPlainTextEdit"/>
        </item>
       </layout>
      </item>
      <item row="0" column="0" colspan="2">
       <layout class="QGridLayout" name="nameLayout">
        <item row="0" column="0">
         <widget class="QLabel" name="urlLabel">
          <property name="text">
           <string>RSS-Feed-URL:</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLineEdit" name="urlLineEdit"/>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="nameLabel">
          <property name="font">
           <font>
            <pointsize>11</pointsize>
            <italic>true</italic>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>Name:</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QLineEdit" name="nameLineEdit"/>
        </item>
       </layout>
      </item>
      <item row="3" column="1">
       <widget class="QGroupBox" name="flagGroupBox">
        <property name="title">
         <string>Misc</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="QCheckBox" name="activatedCheckBox">
           <property name="text">
            <string>Activated</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="useInfoFromFeedCheckBox">
           <property name="text">
            <string>Use name and description from feed</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="embedImagesCheckBox">
           <property name="text">
            <string>Embed images</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="saveCompletePageCheckBox">
           <property name="text">
            <string>Save complete web page (experimental for local feeds)</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="5" column="1">
       <widget class="QGroupBox" name="groupBox">
        <property name="title">
         <string>Transformation</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_4">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout">
           <item>
            <widget class="QLabel" name="transformationTypeLabel">
             <property name="text">
              <string>Transformation type</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="previewButton">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="text">
              <string>Preview &amp;&amp; Transformation</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
      <item row="3" column="0" rowspan="3">
       <widget class="QGroupBox" name="typeGroupBox">
        <property name="title">
         <string>Type</string>
        </property>
        <layout class="QGridLayout" name="gridLayout_2">
         <property name="leftMargin">
          <number>6</number>
         </property>
         <property name="topMargin">
          <number>6</number>
         </property>
         <property name="rightMargin">
          <number>6</number>
         </property>
         <property name="bottomMargin">
          <number>6</number>
         </property>
         <item row="0" column="0">
          <widget class="QCheckBox" name="typeForumCheckBox">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>Forum</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="RSComboBox" name="forumComboBox"/>
         </item>
         <item row="8" column="0">
          <widget class="QCheckBox" name="typeLocalCheckBox">
           <property name="text">
            <string>Local Feed</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <item>
            <widget class="QCheckBox" name="updateForumInfoCheckBox">
             <property name="text">
              <string>Update forum information</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item row="4" column="1">
          <widget class="RSComboBox" name="postedComboBox"/>
         </item>
         <item row="4" column="0">
          <widget class="QCheckBox" name="typePostedCheckBox">
           <property name="text">
            <string>Board</string>
           </property>
          </widget>
         </item>
         <item row="6" column="1">
          <layout class="QGridLayout" name="gridLayout">
           <item row="0" column="0">
            <widget class="QCheckBox" name="updatePostedInfoCheckBox">
             <property name="text">
              <string>Update board information</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QCheckBox" name="postedOnlyImageCheckBox">
             <property name="text">
              <string>Only image</string>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QCheckBox" name="postedFirstImageCheckBox">
             <property name="text">
              <string>Use first image as board image</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QCheckBox" name="postedShinkImageCheckBox">
             <property name="text">
              <string>Shrink image</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>HeaderFrame</class>
   <extends>QFrame</extends>
   <header>gui/common/HeaderFrame.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>RSComboBox</class>
   <extends>QComboBox</extends>
   <header>gui/common/RSComboBox.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>urlLineEdit</tabstop>
  <tabstop>nameLineEdit</tabstop>
  <tabstop>descriptionPlainTextEdit</tabstop>
  <tabstop>typeForumCheckBox</tabstop>
  <tabstop>forumComboBox</tabstop>
  <tabstop>updateForumInfoCheckBox</tabstop>
  <tabstop>typePostedCheckBox</tabstop>
  <tabstop>postedComboBox</tabstop>
  <tabstop>updatePostedInfoCheckBox</tabstop>
  <tabstop>postedShinkImageCheckBox</tabstop>
  <tabstop>postedFirstImageCheckBox</tabstop>
  <tabstop>postedOnlyImageCheckBox</tabstop>
  <tabstop>typeLocalCheckBox</tabstop>
  <tabstop>activatedCheckBox</tabstop>
  <tabstop>useInfoFromFeedCheckBox</tabstop>
  <tabstop>embedImagesCheckBox</tabstop>
  <tabstop>saveCompletePageCheckBox</tabstop>
  <tabstop>previewButton</tabstop>
  <tabstop>useAuthenticationCheckBox</tabstop>
  <tabstop>userLineEdit</tabstop>
  <tabstop>passwordLineEdit</tabstop>
  <tabstop>useStandardStorageTimeCheckBox</tabstop>
  <tabstop>clearCachePushButton</tabstop>
  <tabstop>storageTimeSpinBox</tabstop>
  <tabstop>useStandardUpdateInterval</tabstop>
  <tabstop>updateIntervalSpinBox</tabstop>
  <tabstop>useStandardProxyCheckBox</tabstop>
  <tabstop>proxyAddressLineEdit</tabstop>
  <tabstop>proxyPortSpinBox</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
